.zenodo.json

```
@@ -1,15 +1,63 @@
 {
   "title": "xsdba",
   "creators": [
+    {
+      "name": "Dupuis, Éric",
+      "affiliation": "Ouranos, Montréal, Québec, Canada",
+      "orcid": "0000-0001-7976-4596"
+    },
+    {
+      "name": "Bourgault, Pascal",
+      "affiliation": "Ouranos, Montréal, Québec, Canada",
+      "orcid": "0000-0003-1192-0403"
+    },
```

Alphabetical AND follows your LOC tier list 
```suggestion
    {
      "name": "Bourgault, Pascal",
      "affiliation": "Ouranos, Montréal, Québec, Canada",
      "orcid": "0000-0003-1192-0403"
    },
        {
      "name": "Dupuis, Éric",
      "affiliation": "Ouranos, Montréal, Québec, Canada",
      "orcid": "0000-0001-7976-4596"
    },
```